### PR TITLE
Investigate errors with new --prune tests

### PR DESCRIPTION
**Pull request by jaimergp**
### Description

https://github.com/conda/conda/pull/9614 introduced new tests for `--prune` which are now failing here. Let's investigate what's going on. We'll either fix or skip/xfail to keep CI in the other PRs.

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

File: conda_libmamba_solver/__init__.py [hunk 1]
```diff
@@ -29,3 +29,6 @@ def get_solver_class(key="libmamba"):
         )
         return LibMambaSolver
     raise ValueError("Key must be 'libmamba'")
+
+
+# TRIGGER CI; REMOVE
```
Comment (jezdez) [inline conda_libmamba_solver/__init__.py:34]: Reminder ^

File: dev/collect_upstream_conda_tests/collect_upstream_conda_tests.py [hunk 2]
```diff
@@ -25,6 +25,10 @@
         # Features / nomkl involved
         "test_features_solve_1",
         "test_prune_1",
+        "test_update_prune_2",
+        "test_update_prune_3",
+        # Message expected, but libmamba does not report constraints
+        "test_update_prune_5",
```
Comment (jezdez) [inline dev/collect_upstream_conda_tests/collect_upstream_conda_tests.py:31]: Is this something we can tweak in the upstream test to work better here?

**Review by jezdez — COMMENTED** [review]
Not sure if the test errors are related or not

File: dev/collect_upstream_conda_tests/collect_upstream_conda_tests.py (hunk 2 above)
Comment (jaimergp) [reply dev/collect_upstream_conda_tests/collect_upstream_conda_tests.py:31]: Not really. libmamba's implementation doesn't really report that kind of thing so if anything we have to add more messaging to conda-libmamba-solver wrapping logic.

**Review by jezdez — APPROVED** [review]

**Comment by jaimergp** [comment]
Will be superseded by https://github.com/conda/conda-libmamba-solver/pull/270

**Pull request closed without merging**